scripts/killall_sglang.sh

```diff
@@ -1,5 +1,7 @@
 # Kill all SGLang processes and free the GPU memory.
 
+nvidia-smi
 kill -9 $(ps aux | grep 'multiprocessing.spawn' | grep -v 'grep' | awk '{print $2}')
 kill -9 $(ps aux | grep 'sglang.launch_server' | grep -v 'grep' | awk '{print $2}')
 kill -9 $(ps aux | grep 'sglang.bench' | grep -v 'grep' | awk '{print $2}')
```

Comment:
any reasons why the existing one is not sufficient?

Reply:
This PR is not meant for merging, it is for verification because the issue still exists. Through nvidia-smi, it can be seen that the current user does not have permission to kill those two processes occupying GPU memory.

Reply:
It seems that the machine was set to shared mode when scheduled by k8s, allowing other instances to use GPU resources. The user running CI currently does not have permission to reclaim them.

Reply:
And there is no problem with merging this PR. We can run an nvidia-smi to check the situation before running it. I am also quite curious about how to better solve this problem.

Reply:
I've update the scripts. Compared to before, it will print the result of nvidia-smi for troubleshooting when there are issues, and an option to kill all has been added, just set the parameter great 0.